### DISCUSSION TITLE
Is on-line chat working?

### DISCUSSION THREAD

**Discussion by lutzbickhardt** [toitlang/toit#344] [Q&A] [answered]
I always get: failes to send. Retry?
(is this the proper place to report things like this?)

**Comment by floitsch** [comment, marked as answer]
Already a few days old, but was this for the chat on toit.io ?
If yes, it should work.

**Comment by lutzbickhardt** [comment]
?it was out when i posted, but came up later; had a good chat then ...


Cordialement

--------------------------
Lutz BICKHARDT
***@***.***


Graumannsweg 42, D-22087 Hamburg, Germany

SKYPE: bickhardt

+49 162 9784296
+49 172 3846086
+49 40 2275812

________________________________
De : Florian Loitsch ***@***.***>
Envoyé : lundi 31 janvier 2022 16:54
À : toitlang/toit
Cc : Bickhardt, Lutz; Author
Objet : Re: [toitlang/toit] Is on-line chat working? (Discussion #344)


Already a few days old, but was this for the chat on toit.io ?
If yes, it should work.

-
Reply to this email directly, view it on GitHub<https://github.com/toitlang/toit/discussions/344#discussioncomment-2082534>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAH5UNVK3J6RYJDUMJ6ELC3UY2WE5ANCNFSM5MTHHTXQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
